### PR TITLE
files with assignment

### PR DESCRIPTION
**Pull request by SiddhiBAug19**

### CODE REVIEW

File: src/siddhiCoreJava/LargestOfThree.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	public static void main(String[] args) 
+	
+	{
+			int a = 13;
+			int b = 17;
+			int c =9;
+			
+			if (a>b && a>c)
+			{
+				System.out.println("a is greater");
```
Comment (ghost) [inline src/siddhiCoreJava/LargestOfThree.java:14]: it should be like below code
System.out.println(a+ " is greater");

File: src/siddhiCoreJava/LeapYear.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package siddhiCoreJava;
+
+import java.util.Scanner;
+
+public class LeapYear {
+	
+	static void leapyear(int y)
+	{
+		if (y%4 ==0 && y%400 ==0)
```
Comment (ghost) [inline src/siddhiCoreJava/LeapYear.java:9]: here it should be OR condition

File: src/siddhiCoreJava/WeekdaySwitch.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package siddhiCoreJava;
+
+import java.util.Scanner;
+
+public class WeekdaySwitch {
+
+	public static void main(String[] args) 
+	
+	{
```
Comment (ghost) [inline src/siddhiCoreJava/WeekdaySwitch.java:9]: what if i wanted know day on 23rd date
how you will handle this condition
think on user given date and 7

some reminder value think..

**Review by ghost — APPROVED** [review]
do changes in next PR
other code looks good to me
some java file related to Test and some are blank

File: src/siddhiCoreJava/LargestOfThree.java (hunk 1 above)
Comment (SiddhiBAug19) [reply src/siddhiCoreJava/LargestOfThree.java:14]: Made the required changes

**Pull request merged**